### ISSUE TITLE
Unexpected value 'PEND' by WalletApi

### ISSUE THREAD

**Issue by wltj920** [gateio/gateapi-java#7]
When I use WallApi.listDeposits, throw an exception like this:

```shell
Exception in thread "pool-5-thread-30" java.lang.IllegalArgumentException: Unexpected value 'PEND'
	at io.gate.gateapi.models.LedgerRecord$StatusEnum.fromValue(LedgerRecord.java:94)
	at io.gate.gateapi.models.LedgerRecord$StatusEnum$Adapter.read(LedgerRecord.java:106)
	at io.gate.gateapi.models.LedgerRecord$StatusEnum$Adapter.read(LedgerRecord.java:97)
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:897)
	at com.google.gson.Gson.fromJson(Gson.java:846)
	at io.gate.gateapi.JSON.deserialize(JSON.java:142)
	at io.gate.gateapi.ApiClient.deserialize(ApiClient.java:781)
	at io.gate.gateapi.ApiClient.handleResponse(ApiClient.java:984)
	at io.gate.gateapi.ApiClient.execute(ApiClient.java:911)
	at io.gate.gateapi.api.WalletApi.listDepositsWithHttpInfo(WalletApi.java:430)
	at io.gate.gateapi.api.WalletApi.access$500(WalletApi.java:38)
	at io.gate.gateapi.api.WalletApi$APIlistDepositsRequest.execute(WalletApi.java:526)
	at com.qkbstudio.qkminer.module.qkm.service.ex.rest.impl.GateIoRestServiceImpl.getDepositRecords(GateIoRestServiceImpl.java:121)
	at com.qkbstudio.qkminer.module.qkm.job.DepositListenerJob.lambda$executeInternal$0(DepositListenerJob.java:78)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

A new `StausEnum` founded but sdk not updated

**Comment by revilwang** [comment]
Thanks for reporting. We'll update it ASAP

**Comment by revilwang** [comment]
Fix in new version 6.23.0.
